File: uri/area_do_circulo/1002.go
package main

import (
	"C"
	"fmt"
)

func areaCalc(raio float64) float64 {
	var pi float64 = 3.14159
	return (pi * raio * raio)
}

func main1002() {
	var raio, area float64
	fmt.Scanf("%g", &raio)
	area = areaCalc(raio)
	fmt.Printf("A=%.4f\n", area)
}

func main() {
	main1002()
}

File: uri/area_do_circulo/1002.c
#include <stdio.h>

double calcularea(double raio);

int main(){
	
    double rad, area;
    scanf("%lf", &rad);
    area = calcularea(rad);
    printf("A=%.4lf\n", area);
    return 0;
}

double calcularea(double raio)
{
    double pi = 3.14159;
    return (pi * raio * raio);    
}
